api_server.py

```diff
@@ -66,6 +67,7 @@ async def manage_teleop_startup(scope):
     if manual_controller is not None:
         manual_controller.stop_teleop()
     manual_controller = ManualController(session, scope, use_bridge_ros2dds)
+    time.sleep(1)
```

Comment:
Is there any reason why we need to sleep here? Maybe you can add some comments.

Reply:
I removed sleep in [this commit](https://github.com/evshary/zenoh_autoware_fms/pull/29/commits/729a0455058c203fcd2a4d4f12763a986c036f5f), sorry for the inconvenience.

Reply:
No worries. We're always open to discussion :smile: